app/Project.php

```diff
@@ -54,6 +58,35 @@ public function getIsBehindLatestAttribute()
         return version_compare($this->desired_laravel_version, $this->current_laravel_version) > 0;
     }
 
+    public function getStatusAttribute()
+    {
+        if (!$this->is_behind_latest) {
+            return self::STATUS_CURRENT;
+        }
+
+        return $this->hasSecurityFixes() ? self::STATUS_BEHIND : self::STATUS_INSECURE;
+    }
+
+    private function hasSecurityFixes()
+    {
+        $major= explode('.', $this->current_laravel_version)[0];
+
+        if ($major === '8') {
+            // @see https://laravel.com/docs/8.x/releases#support-policy
+            return strtotime('September 8th, 2021') >= strtotime('today');
+        }
+        if ($major === '7') {
```

Comment:
Styling: Please add a space above this line

Reply:
Done: https://github.com/tighten/checkmate/pull/31/commits/a749aa1f8b8e8649bc9ff247894d6161e2aa2d37